creators_custom/StopsCreatorFenix.py

```diff
@@ -138,25 +217,6 @@ def get_interim_stop_name(self):
         self.name = winner.tags["name"].encode('utf-8')
 
     @staticmethod
-    def from_relation_member(member):
-        stop_id = member.ref
-
-        if isinstance(member, overpy.RelationNode):
-            stop_type = "node"
-        elif isinstance(member, overpy.RelationWay):
-            stop_type = "way"
-        else:
-            raise RuntimeError("Unknown type: " + str(member))
-
-        return Stop(stop_id, stop_type)
-
-    @staticmethod
-    def normalize_name(old_name):
-        name = STOP_REGEX.sub(r'\1', old_name)
-        name = name.replace('Terminal de Integração da Lagoa da Conceição', 'TILAG')
-        return name
-
```

Comment:
This method was moved in the Route class because it gets only called from there. This was easier than doing a lot of cross imports of classes that otherwise do not interfere.